internal/testutils/helper.go

```diff
@@ -120,4 +99,30 @@ func ValidateImportBlocks(t *testing.T, resource connector.ExportableResource, e
 		}
 		resourceIDs[importBlock.ResourceID] = true
 	}
+
+	expectedImportBlocksMap := map[string]connector.ImportBlock{}
+	for _, importBlock := range *expectedImportBlocks {
+		expectedImportBlocksMap[importBlock.ResourceName] = importBlock
+	}
+
+	// Check number of export blocks
+	expectedNumberOfBlocks := len(expectedImportBlocksMap)
+	actualNumberOfBlocks := len(*importBlocks)
+	if actualNumberOfBlocks != expectedNumberOfBlocks {
+		t.Fatalf("Expected %d import blocks, got %d", expectedNumberOfBlocks, actualNumberOfBlocks)
+	}
+
+	// Make sure the importblocks match the expected import blocks
+	for _, importBlock := range *importBlocks {
+		expectedImportBlock := (expectedImportBlocksMap)[importBlock.ResourceName]
+
+		if expectedImportBlock.Equals(connector.ImportBlock{}) {
```

Comment:
I'd use the `ok` response to check if nothing was returned, rather than checking for the zero value here.
```
expectedImportBlock, ok := ...
if ! ok {
  // if wasn't in the map
}
```

Reply:
I didn't know about this! Fixed.